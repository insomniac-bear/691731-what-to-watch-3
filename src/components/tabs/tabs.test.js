import React from 'react';
import renderer from 'react-test-renderer';
import Tabs from './tabs.jsx';

const mockData = {
  id: 0,
  filmName: `Film Named`,
  posterUrl: `someUrl`,
  genre: `Genre`,
  release: 2018,
  runtime: 99,
  rating: 9.9,
  describe: `Some discribe`,
  director: `Director`,
  actors: `Actor, Actor, Actor`,
  comments: [
    {
      author: `Author name`,
      rating: 9.9,
      date: `January 01, 2001`,
      textComment: `Some comment`,
    },
    {
      author: `Author name`,
      rating: 9.9,
      date: `January 01, 2001`,
      textComment: `Some comment`,
    },
    {
      author: `Author name`,
      rating: 9.9,
      date: `January 01, 2001`,
      textComment: `Some comment`,
    }
  ],
};

it(`Render TabsPage`, () => {
  const tree = renderer
    .create(<Tabs
      filmData={mockData}
      activeTab={`Overview`}
      renderTab={jest.fn()}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
