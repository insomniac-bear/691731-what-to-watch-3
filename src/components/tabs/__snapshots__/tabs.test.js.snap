// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render TabsPage 1`] = `
<div
  className="movie-card__desc"
>
  <nav
    className="movie-nav movie-card__nav"
  >
    <ul
      className="movie-nav__list"
    />
  </nav>
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      99
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Avesome
      </span>
      <span
        className="movie-rating__count"
      >
        240 ratings
      </span>
    </p>
  </div>
  <div
    className="movie-card__text"
  >
    <p>
      Some description
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Director
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Actor Actor
      </strong>
    </p>
  </div>
</div>
`;
